docs/optimization_guide/nncf/ptq/code/ptq_torch_fx.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#! [dataset]
+import nncf
+import torch
+
+calibration_loader = torch.utils.data.DataLoader(...)
+
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+calibration_dataset = nncf.Dataset(calibration_loader, transform_fn)
+#! [dataset]
+
+#! [quantization]
+import torchvision
+import openvino.torch
+model = torchvision.models.resnet50(pretrained=True)
+
+input_fp32 = ... # FP32 model input
+
+with nncf.torch.disable_patching():
+    exported_model = torch.export.export_for_training(model, args=(input_fp32,)).module()
+    quantized_model = nncf.quantize(exported_model, calibration_dataset)
```

Comment:
what is a format of `quantized_model`? torch.nn.Module?

Reply:
The format is the [ExportedProgram](https://pytorch.org/docs/stable/export.html#torch.export.ExportedProgram)